src/components/v5/common/ActionSidebar/partials/forms/SplitPaymentForm/SplitPaymentForm.tsx

```diff
@@ -0,0 +1,117 @@
+import React, { FC } from 'react';
+
+import { FormattedMessage } from 'react-intl';
+import { useSplitPayment } from './hooks';
+import { ActionFormBaseProps } from '../../../types';
+import ActionFormRow from '~v5/common/ActionFormRow';
+import TeamsSelect from '~v5/common/ActionSidebar/partials/TeamsSelect';
+import AmountField from '~v5/common/ActionSidebar/partials/AmountField';
+import DescriptionField from '~v5/common/ActionSidebar/partials/DescriptionField';
+import { FormCardSelect } from '~v5/common/Fields/CardSelect';
+import {
+  DECISION_METHOD_OPTIONS,
+  DISTRIBUTION_METHOD_OPTIONS,
+} from '../../consts';
+import { formatText } from '~utils/intl';
+import SplitPaymentRecipientsField from './partials/SplitPaymentRecipientsField';
+
+const displayName = 'v5.common.ActionSidebar.partials.SplitPaymentForm';
+
+const SplitPaymentForm: FC<ActionFormBaseProps> = ({ getFormOptions }) => {
+  const { currentToken, distributionMethod, amount } =
+    useSplitPayment(getFormOptions);
+
+  return (
+    <>
+      <ActionFormRow
+        iconName="pie-chart"
+        fieldName="distributionMethod"
+        tooltip={
+          <FormattedMessage id="actionSidebar.toolip.distributionTypes" />
```

Comment:
please use formatText from from '~utils/intl' everywhere it is possible

Reply:
Done